homeassistant/components/uptime_kuma/binary_sensor.py

```diff
@@ -0,0 +1,42 @@
+"""Summary binary data from Uptime Kuma."""
+from voluptuous.validators import Boolean
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = hass.data[DOMAIN]
+    async_add_entities(
+        UptimeKumaBinarySensor(coordinator, monitor) for monitor in coordinator.data
+    )
+
+
+class UptimeKumaBinarySensor(BinarySensorEntity, CoordinatorEntity):
```

Comment:
Shouldn't this be implementing the `available` property as well? (If the monitored name isn't in the coordinator data anymore?)

Reply:
Doesn't an entity show as unavailable automatically if data is not reported for it?